docs/dashboards/README.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Dashboards overview
+---
+
+# Dashboards overview
+
+![Example dashboard](./images/dashboard.png)
+
+## [Introduction to dashboards](./introduction)
+
+Learn how to arrange tables, charts, and text cards on a dashboard.
+
+## [Dashboard filters](./filters.md)
+
+Add filters to dashboards to update tables and charts.
+
+## [Interactive dashboards](./interactive.md)
+
+Customize what happens when people click on a chart in your dashboard.
+
+## [Dashboard charts with multiple series](./multiple-series.md)
```

Comment:
```suggestion
## [Charts with multiple series](./multiple-series.md)
```

Suggest making this bullet specific to "Charts" rather than "Dashboards", because making a dashboard isn't a prerequisite to making a multi-series chart.

If we have a "Chart" or "Visualization" cluster in the future, we can make a copy of this entry point there :)